crates/omnix-cli/src/command/show.rs

```diff
@@ -115,7 +119,51 @@ impl ShowConfig {
         FlakeOutputTable {
             rows: Row::vec_from_btreemap(flake.schema.nixos_configurations),
             title: "🐧 NixOS Configurations".to_string(),
-            command: "nixos-rebuild switch --flake .#<name>".to_string(),
+            command: format!("nixos-rebuild switch --flake {}#<name>", self.flake_url),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.darwin_configurations),
+            title: "🍏 Darwin Configurations".to_string(),
+            command: format!("darwin-rebuild switch --flake {}#<name>", self.flake_url),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.nixos_modules),
+            title: "🔧 NixOS Modules".to_string(),
+            // TODO: Command should be optional
+            command: "".to_string(),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.docker_images),
+            title: "🐳 Docker Images".to_string(),
+            // TODO: Try if the below command works
+            command: format!("nix build {}#dockerImages.<name>", self.flake_url),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.overlays),
+            title: "🎨 Overlays".to_string(),
+            command: "".to_string(),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.templates),
+            title: "📝 Templates".to_string(),
+            command: format!("nix flake init -t {}#<name>", self.flake_url),
+        }
+        .print();
+
+        FlakeOutputTable {
+            rows: Row::vec_from_btreemap(flake.schema.schemas),
+            title: "📜 Schemas".to_string(),
+            command: "".to_string(),
```

Comment:
We should generally use `Option<String>` rather than overloading `String` with empty strings. It would also allow us to avoid empty parentheses like in NixOS Modules here,

<img width="291" alt="image" src="https://github.com/user-attachments/assets/d9414b2a-d74f-4304-8a1e-75ac4769cdf4">

Reply:
https://github.com/juspay/omnix/pull/166/commits/f5b597b6c6dba61e32b60d68ac582c9d1f7769f1